analysis/src/ModuleResolution.ml

```diff
@@ -1,5 +1,5 @@
 let rec resolveNodeModulePath ~startPath name =
   let path = Filename.concat (Filename.concat startPath "node_modules") name in
   if Files.exists path then Some path
-  else if startPath = "/" then None
+  else if startPath = "/" || Filename.dirname startPath = startPath then None
```

Comment:
The first condition can be dropped right?

Reply:
Yes it can be dropped.